tests/dummy/app/styles/app.scss

```diff
@@ -22,4 +22,16 @@
 
 .demo-flex {
   flex: 1;
+
+  .frost-chart-x-axis-svg {
```

Comment:
Issue: make a blockless form of the svg based axis more explicit (label, should have grid line or no, alignment, etc.) so that we can calculate the position and save people the headache of having to do this themselves